src/main/java/carRacing/model/RandomMoveStrategy.java

```diff
@@ -1,18 +1,15 @@
 package carRacing.model;
 
 public class RandomMoveStrategy implements MoveStrategy {
```

Comment:
람다를 사용하고 있으니 명시적으로 선언하면 좋겠네요
```suggestion
@FunctionalInterface
public class RandomMoveStrategy implements MoveStrategy {
```